try{

}catch(Exception erro){
    System.out.println(erro);
}finally{

}
